docs/ref/class-based-views/mixins-editing.txt

```diff
@@ -97,7 +97,7 @@ The following mixins are used to construct Django's editing views:
 
 .. class:: django.views.generic.edit.ModelFormMixin
 
-    A form mixin that works on ``ModelForms``, rather than a standalone form.
+    A form mixin that works on ``ModelForm``\s, rather than a standalone form.
```

Comment:
```suggestion
    A form mixin that provides facilities for creating and displaying ``ModelForm`` instances (rather than a standalone form like ``FormMixin``).
```